src/main/webapp/assets/script.js

```diff
@@ -71,6 +72,8 @@ function onLoad() {
   posts = fetchFakePosts();
   collegeLocation = fetchFakeCollegeLocation();
 
+  addPosts(posts);
+
   // Add the embedded map to the page.
   const script = document.createElement('script');
   script.src = `https://maps.googleapis.com/maps/api/js?key=AIzaSyB_iMemNeUtO8D7kGTmagTgluRlsiFTcDA&callback=initMap`;
```

Comment:
Should this key be exposed?

Reply:
I think Melanie added the Secrets Manager to help with this.

Reply:
All keys have been regenerated and live in the secrets manager now, so that key is invalidated now - #9  has been merged into master so we should be good 👍